.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea
+__pycache__
+project.lock
```

Comment:
In several places, final lines are missing: I think in spaCy we normally include them even in such files that can't be formatted by `black` or `prettier`?

Reply:
Thanks, wasn't aware. I added `setup.cfg` and `pre-commit.yaml` and formatted everything with `black`. Which other files are you missing the last newline in?

Reply:
`*.yaml` and `.gitignore` files. See the red "stop" symbols under https://github.com/explosion/wikid/pull/2/files.